docs/sources/next/javascript-api/k6-experimental/streams/readablestream/_index.md

```diff
@@ -0,0 +1,223 @@
+---
+title: 'ReadableStream'
+description: 'ReadableStream represents a readable stream of data.'
+weight: 10
+---
+
+# ReadableStream
+
+The `ReadableStream` type represents a readable stream of data.
+
+## Constructing a ReadableStream
+
+The constructor creates a new `ReadableStream` object.
+
+It takes two **optional** arguments, `underlyingsource` allowing to define the underlying source of data, as well as the queuing strategy to adopt.
```

Comment:
```suggestion
It takes two _optional_ arguments:

- `underlyingsource`: defines the underlying source of data.
- `queuingStrategy`: the queuing strategy to adopt.
```

We generally use _italics_ for emphasis and **bold** for UI items in the Grafana style guide.